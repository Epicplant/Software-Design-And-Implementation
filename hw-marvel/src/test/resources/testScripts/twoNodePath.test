# Tests correct path if there is exactly two edges edge we need to travel to get from start node to end node

# Loads a new graph from a csv file
LoadGraph heroes maxPathTwoHeroGraph.csv

# Finds shortest path between heroes
FindPath heroes A C