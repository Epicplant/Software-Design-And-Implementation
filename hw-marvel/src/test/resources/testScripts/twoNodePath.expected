# Tests correct path if there is exactly two edges edge we need to travel to get from start node to end node

# Loads a new graph from a csv file
loaded graph heroes

# Finds shortest path between heroes
path from A to C:
A to B via 1
B to C via 2